### DISCUSSION TITLE
Able to stream  to ChromeCast

### DISCUSSION THREAD

**Discussion by TFL999** [f1multiviewer/issue-tracker#124] [Feature requests]
It would be awesome to have the ability to cast one stream to the ChromeCast and have this synced with the other streams. 
Most often I watch F1 in the living room with the rest of the family, That way it is easy to have the international stream on the tv (with Grand Prix Radio over the sound bar) and I can have my laptop with extra screen in front of me with the driver cams and telemetry/RCM screen.

**Comment by f1multiviewer** [comment]
This is a duplicate of https://github.com/f1multiviewer/issue-tracker/discussions/52
